src/main/java/io/druid/data/input/MapBasedRow.java

```diff
@@ -159,6 +161,52 @@ public long getLongMetric(String metric)
   }
 
   @Override
+  public int getIntMetric(String metric)
+  {
+    Object metricValue = event.get(metric);
+
+    if (metricValue == null) {
+      return 0;
+    }
+
+    if (metricValue instanceof Number) {
+      return ((Number) metricValue).intValue();
+    } else if (metricValue instanceof String) {
+      try {
+        return Integer.valueOf(((String) metricValue).replace(",", ""));
```

Comment:
it's definitely caused issues (I remember someone writing in on the lists a while ago about their numbers with ',' decimal points mysteriously parsing as much larger numbers), but this PR is probably not the right place to change that.